h/tasks/indexer.py

```diff
@@ -43,7 +43,11 @@ def delete_annotation(id_):
 @celery.task(acks_late=False)
 def sync_annotations(limit):
     search_index = celery.request.find_service(name="search_index")
-    search_index.sync(limit)
+
+    counts = search_index.sync(limit)
+
+    log.info(dict(counts))
+    newrelic.agent.record_custom_metrics(counts.items())
```

Comment:
This actually reports _all_ of the counts from `Queue.sync()` to New Relic, although I'm only really planning on using the `TOTAL_` ones

Reply:
Realising that this is gonna report log messages to New Relic, I need to replace them with New Relic-style metric names :/